x/savings/keeper/keeper.go

```diff
@@ -0,0 +1,42 @@
+package keeper
+
+import (
+	"fmt"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
+	"github.com/tendermint/tendermint/libs/log"
+
+	"github.com/kava-labs/kava/x/savings/types"
+)
+
+// Keeper struct for pricefeed module
```

Comment:
looks like some comments needs to be updated

Reply:
fixed in https://github.com/Kava-Labs/kava/pull/1188/commits/3d2082e21d168df30ca4105b160bc9b4378b1c08